spec.md

```diff
@@ -718,9 +719,10 @@ $ curl 'http://username:password@broker-url/v2/service_instances/:instance_id?ac
 
 | Status Code | Description |
 | --- | --- |
-| 200 OK | Service instance was deleted. The expected response body is `{}`. |
-| 202 Accepted | Service instance deletion is in progress. This triggers the marketplace to poll the [Last Operation](#polling-last-operation) endpoint for operation status. |
-| 410 Gone | MUST be returned if the service instance does not exist. The expected response body is `{}`. |
+| 200 OK | MUST be returned if the service instance was deleted as a result of this request. The expected response body is `{}`. |
+| 202 Accepted | MUST be returned if the service instance deletion is in progress. This triggers the marketplace to poll the [Service Instance Last Operation Endpoint](#polling-last-operation) for operation status. Note that a resent `DELETE` request MUST return a `202 Accepted`, not a `200 OK`, if the delete request has not completed yet. |
+| 404 Not Found | MUST be returned if the service instance does not exist. |
+| 410 Gone | MAY be returned if the service instance is known to the broker but had been previously deleted. The expected response body is `{}`. While a `410 Gone` is allowed, the RECOMMENDED response is `404 Not Found`. |
```

Comment:
Same issues with 404. We should not introduce this as a supported response without further discussion.